node_tree.py

```diff
@@ -403,13 +368,6 @@ def process(self):
 
         self.has_changed = False
 
-    def get_nodes_supporting_draft_mode(self):
-        draft_nodes = []
-        for node in self.nodes:
-            if hasattr(node, 'does_support_draft_mode') and node.does_support_draft_mode():
-                draft_nodes.append(node)
-        return draft_nodes
-
```

Comment:
I'm not sure that this is not used. Did you check?

Reply:
Everything what I was removing I surely checked.

Reply:
![2020-08-25_21-37-30](https://user-images.githubusercontent.com/28003269/91208389-32c31380-e71b-11ea-9219-74dff3106c1f.png)